doc/mgmt/SONiC_OC_MGMT_Interface_HLD.md

```diff
@@ -0,0 +1,259 @@
+          |  +--ro mtu?             uint16
+          |  +--ro description?     string
+          |  +--ro enabled?         boolean
+          |  +--ro ifindex?         uint32
+          |  +--ro admin-status     enumeration
+          |  +--ro oper-status      enumeration
+          |  +--ro counters
+          |  |  +--ro in-octets?             oc-yang:counter64
+          |  |  +--ro in-pkts?               oc-yang:counter64
+          |  |  +--ro in-unicast-pkts?       oc-yang:counter64
+          |  |  +--ro in-broadcast-pkts?     oc-yang:counter64
+          |  |  +--ro in-multicast-pkts?     oc-yang:counter64
+          |  |  +--ro in-discards?           oc-yang:counter64
+          |  |  +--ro in-errors?             oc-yang:counter64
+          |  |  +--ro out-octets?            oc-yang:counter64
+          |  |  +--ro out-pkts?              oc-yang:counter64
+          |  |  +--ro out-unicast-pkts?      oc-yang:counter64
+          |  |  +--ro out-broadcast-pkts?    oc-yang:counter64
+          |  |  +--ro out-multicast-pkts?    oc-yang:counter64
+          |  |  +--ro out-discards?          oc-yang:counter64
+          |  |  +--ro out-errors?            oc-yang:counter64
+          +--rw subinterfaces
+          |  +--rw subinterface* [index]
+          |     +--rw index           -> ../config/index
+          |     +--rw oc-ip:ipv4
+          |     |  +--rw oc-ip:addresses
+          |     |  |  +--rw oc-ip:address* [ip]
+          |     |  |     +--rw oc-ip:ip        -> ../config/ip
+          |     |  |     +--rw oc-ip:config
+          |     |  |     |  +--rw oc-ip:ip?              oc-inet:ipv4-address
+          |     |  |     |  +--rw oc-ip:prefix-length?   uint8
+          |     |  |     +--ro oc-ip:state
+          |     |  |     |  +--ro oc-ip:ip?              oc-inet:ipv4-address
+          |     |  |     |  +--ro oc-ip:prefix-length?   uint8
+          |     +--rw oc-ip:ipv6
+          |        +--rw oc-ip:addresses
+          |        |  +--rw oc-ip:address* [ip]
+          |        |     +--rw oc-ip:ip        -> ../config/ip
+          |        |     +--rw oc-ip:config
+          |        |     |  +--rw oc-ip:ip?              oc-inet:ipv6-address
+          |        |     |  +--rw oc-ip:prefix-length    uint8
+          |        |     +--ro oc-ip:state
+          |        |     |  +--ro oc-ip:ip?              oc-inet:ipv6-address
+          |        |     |  +--ro oc-ip:prefix-length    uint8
+          +--rw oc-eth:ethernet
+          |  +--rw oc-eth:config
+          |  |  +--rw oc-eth:auto-negotiate?        boolean
+          |  |  +--rw oc-eth:port-speed?            identityref
+          |  +--ro oc-eth:state
+          |  |  +--rw oc-eth:auto-negotiate?        boolean
+          |  |  +--ro oc-eth:port-speed?               identityref
+
+
+### 3.6.2 CLI
+#### 3.6.2.1 Configuration Commands
+1. interface Management <Interface Id>
+  CLI -> Interface syntax and output
+  sonic(config)# interface
```

Comment:
Enclose these CLI examples within 3 backticks to get monospaced formatting \`\`\`
```
like this
```

Reply:
Addressed